src/Symfony/Component/Console/Input/InputOption.php

```diff
@@ -173,13 +173,22 @@ public function isArray(): bool
         return self::VALUE_IS_ARRAY === (self::VALUE_IS_ARRAY & $this->mode);
     }
 
+    /**
+     * Returns true if the option may have either a positive or negative value.
```

Comment:
Reading "negative value" I understand that I can pass a negative int (i.e. something like `-5`, that is not supported by the input parser).
That's an other feature described in this blog post: https://symfony.com/blog/new-in-symfony-5-3-negatable-command-options

Reply:
I copied it directly from the `VALUE_NEGATABLE` docblock. I've now rephrased both to be more specific and clear in my opinion; please let me know if you have other ideas about the phrasing.